<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgUi</class>
 <widget class="QDialog" name="CQDlgUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGl-based Custom UIs</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Selected OpenGl-based custom UI</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqAddNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Add new user interface</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Main properties</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal / vertical cell count</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqCellCountH">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqCellCountV">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>20</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqCellSizeV">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>40</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal / vertical cell size [pt]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqCellSizeH">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqUiPosH">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Horizontal / vertical UI position</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqUiPosV">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>60</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqUiBorderL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is relative to left border</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqUiBorderT">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>171</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is relative to top border</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqVisible">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is visible</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqVisibleDuringSimulation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>201</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is visible only during simulation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqVisibleWhenObjectSelected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>301</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is visible if associated object is selected</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqSelectObject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>281</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Select associated object if UI is clicked</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqMoveable">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>UI is moveable</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="qqFixedWidthFont">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Use fixed-width font</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Display UI in</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqPageCombo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>131</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>UI is associated with</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqAssociatedObject">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Other</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QToolButton" name="qqAdjustRolledUpSizes">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Adjust rolled-up sizes</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqSetTexture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Set UI background texture</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAddNew</tabstop>
  <tabstop>qqCellCountH</tabstop>
  <tabstop>qqCellCountV</tabstop>
  <tabstop>qqCellSizeH</tabstop>
  <tabstop>qqCellSizeV</tabstop>
  <tabstop>qqUiPosH</tabstop>
  <tabstop>qqUiPosV</tabstop>
  <tabstop>qqUiBorderL</tabstop>
  <tabstop>qqUiBorderT</tabstop>
  <tabstop>qqVisible</tabstop>
  <tabstop>qqMoveable</tabstop>
  <tabstop>qqVisibleDuringSimulation</tabstop>
  <tabstop>qqFixedWidthFont</tabstop>
  <tabstop>qqVisibleWhenObjectSelected</tabstop>
  <tabstop>qqSelectObject</tabstop>
  <tabstop>qqPageCombo</tabstop>
  <tabstop>qqAssociatedObject</tabstop>
  <tabstop>qqSetTexture</tabstop>
  <tabstop>qqAdjustRolledUpSizes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
